docs/overview.md

```diff
@@ -0,0 +1,91 @@
+
+### Fees
+The main incentive for becoming a Validator is the service fee paid by users. The fee is a percentage of the transferred amount, paid on the native asset.
+For example transferring 100 HBAR from Hedera to the EVM chain (WHBAR) is going to have 1% service fee (1 HBAR) transferred to the Bridge Validators.
+
+*Note: The Service fee is configurable property and determined by the validators*
+
+### Transferring HBAR/HTS from Hedera to the EVM chain
+
+The transfer of assets from Hedera to the EVM chain is described in the following sequence diagram.
+<p align="center">
+  <img src="./assets/hedera-to-evm.png">
+</p>
+
+#### Steps
+1. **Initiating the transfer**
+   Alice wants to transfer `HBAR`/`HTS token` from Hedera to the EVM chain. She opens any UI that integrates the Bridge and sends the asset to the `Bridge` Account. The memo of the transfer contains the `evm-address` which is going to be the receiver of the wrapped asset on the other chain.
+2. **Picking up the Transfer**
+   The Bridge validator nodes listen for new incoming transfers to the `Bridge` Account. Once they pick up the new transaction, they verify the `state proof` and validate that the `memo` contains a valid EVM address configured as receiver of the wrapped asset.
+3. **Paying out fees**
+
+   **3.1** Each of the Validators create a Schedule Create transaction transferring the `service fee` amount from the `Bridge` account to the list of validators equally *(f.e if the service fee is `7 HBAR` and there are `7` validators, the Schedule Create Transfer will contain Transfer list crediting `1 HBAR` to each of the validators.)*
+
+   **3.2** Due to the nature of Scheduled Transactions, only one will be successfully executed, creating a scheduled Entity and all others will fail with `IDENTICAL_SCHEDULE_ALREADY_CREATED` error, and the transaction receipt will include the `ScheduleID` of the first submitted transaction.
+All validators, except the one that successfully created the Transaction execute `ScheduleSign` and once `n out of m` validators execute the Sign operation, the transfer of the fees will be executed.
+
+4. **Providing Authorisation Signature**
+   Each of the Validators sign the following authorisation message:
+   `{hedera-tx-id}{router-address}{native-token}{receiver}{amount}` using their EVM-compatible private key.
+   The authorisation is then submitted to a topic in Hedera Consensus Service
+
+5. **Waiting for Supermajority**
+   Alice's UI or API waits for a supermajority of the signatures. She can either watch the topic messages stream or fetch the data directly from Validator nodes.
+
+6. **Submitting the EVM Transaction**
+   Once supermajority is reached, Alice submits the transaction to the EVM chain, claiming her wrapped asset. The signature contains the raw data signed in the message: `hedera-tx`, `native-token-address`, `receiver` and the `amount` as well as the set of signatures from the Validators.
```

Comment:
`router-address` is missing
`wrapped-token-address` is signed instead of `native-token-address`

Reply:
You can copy the them from 4.